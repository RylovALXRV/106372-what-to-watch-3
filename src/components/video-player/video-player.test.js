import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const mock = {
  preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  poster: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  isPlaying: true,
  isPause: true,
};

it(`Should VideoPlayer render correctly`, () => {
  const {preview, poster, isPlaying, isPause} = mock;

  const tree = renderer.create(
      <VideoPlayer
        preview={preview}
        poster={poster}
        isPlaying={isPlaying}
        isPause={isPause}
      />, {
        createNodeMock: () => {
          return {};
        }
      }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
