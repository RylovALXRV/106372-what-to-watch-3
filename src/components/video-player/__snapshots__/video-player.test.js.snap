// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  className=""
  height="175"
  width="280"
/>
`;
